src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -1,10 +1,14 @@
 import React, { useState } from "react";
-import StellarSdk, { MemoType } from "stellar-sdk";
+import StellarSdk, { MemoType , FederationServer } from "stellar-sdk";
 import styled from "styled-components";
 import BigNumber from "bignumber.js";
+import { ActionStatus } from "ducks/account";
```

Comment:
An `ActionStatus` enum doesn't seem to be directly tied to an account duck; it seems applicable elsewhere. Seems like a better home would be a file `constants/action_status`?

Reply:
That's a good call. We spoke about that offline but we're going to put that in a constants folder in either of our next PRs. It's just pulled from there now so we don't duplicate code